sphinx/source/index.rst

```diff
@@ -13,3 +13,22 @@ PnPQ
    api/modules
 
 PnPQ is a control library for hardware commonly used in quantum optical experiments (although it is probably useful for classical optical experiments as well!).
+
+Supported devices
+-----------------
+
+**ThorLabs**:
```

Comment:
Continue to use semantic markup for headers, even in nested sections such as this. Avoid literal `:` (colons) in headers.

```suggestion
ThorLabs
^^^^^^^^
```

See https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections